tools/generator-go-sdk/internal/generator/templater_id_parser.go

```diff
@@ -41,7 +41,7 @@ import (
     "strings"
 
     "github.com/hashicorp/go-azure-helpers/resourcemanager/resourceids"
-    "github.com/hashicorp/go-azure-helpers/resourcemanager/resourceids/recaser"
+    "github.com/hashicorp/go-azure-helpers/resourcemanager/recaser"
```

Comment:
_minor_ mind fixing the ordering here?

Reply:
Hadn't noticed that - fixed now! Thanks!